src/services/SendATFReport.ts

```diff
@@ -42,7 +42,12 @@ class SendATFReport {
       }
       this.notifyService = new NotificationService(new NotifyClient(this.apiKey));
     }
-    await this.notifyService.sendNotification(sendNotificationData, response[0].testStationEmails);
+    // VTM allows blank email addresses on a test-station record so check before sending
```

Comment:
I've run and tested this locally and it works as expected, is it worth adding a unit test to prove this logic, and that the console message is logged?

Reply:
Test added